lab2.js

```diff
@@ -48,22 +48,52 @@ function assert(expression, failureMessage) {
  with Dowington.
 */
 
-var hoursSpentInDowington; // TODO: assign me the value of the
+function Blob() {
+}
+
+var blob = new Blob();
+
+var population = 1000;
+var eatingRate = 1;
+var hours = 0;
+var hoursToEat = 0;
+
+Blob.prototype.consumptionRate = function() {
+  while (population > 0) {
```

Comment:
So population is just the variable hanging out above and this method is totally broken without that? When designing objects and methods like this, try to make them as reusable as possible, such that if they were to be exported into someone else's script they would still work.

Here I think it's reasonable to have your Blob constructor be empty but what if we passed population and eatingRate into consumptionRate rather than relying on these global scope variables.